README.md

```diff
@@ -146,6 +146,19 @@ http.Timeout = TimeSpan.FromSeconds(15);
 var client = new FaunaClient("secret", "http://localhost:9090/", httpClient: http);
 ```
 
+#### HTTP 2.0 support
+Starting from version 4.0.0 of this driver (faunadb-csharp), HTTP/2 support is enabled by default for .NET standards 2.1 and above.
+This means that if you use .NET core 3.1 and above (which support that standard), you'll be sending requests to Fauna on HTTP/2. 
+.NET standards lower than 2.1 and .NET frameworks 4.5-4.8 have HTTP/1.1 enabled as the default protocol version, since they lack of support for HTTP/2.
+We've also added an optional parameter if you want to specify the version of the protocol directly:
+```
+var adminClient = new FaunaClient(
+    endpoint: endpoint,
+    secret: secret,
+    httpVersion: HttpVersion.Version20
```

Comment:
as `2.0` is the current default, I think the example could be showing the `1.1`, wdyt?

Reply:
that makes sense, thank you
I've changed to 1.1